### DISCUSSION TITLE
Is GitHub Action only for GitHub Workflows?

### DISCUSSION THREAD

**Discussion by DavideC03** [github/india#82] [Satellite India 2021 Q&A] [answered]
Excuse me,
Can you say me if Others and I can use a GitHub Actions only into GitHub Workflows.

Thanks,
DavideC03

**Comment by shashban** [comment, marked as answer]
GitHub Actions execute workflows on GitHub Runners in response to events. An action can interact with services outside GitHub as well to enhance the E2E workflow.

**Reply by DavideC03** [reply]
Thanks
